### PR TITLE
Add Awesome ChatGPT Prompts

### PR DESCRIPTION
**Pull request by ademidun**
https://github.com/f/awesome-chatgpt-prompts

A collection of prompt examples to be used with the ChatGPT model. It should be included because ChatGPT is a very popular AI Chatbot and the awesome repo linked has [3k stars](https://github.com/f/awesome-chatgpt-prompts/stargazers) despite being [created](https://github.com/f/awesome-chatgpt-prompts/commit/980e43eedfb2cb66297e7e4df120e8d3629c83db) less than 7 days ago.

I know this doesn't meet the rule: **Has been around for at least 30 days.**. However, given the high level of interest in this project, I propose we make an exception.

- PR to add the awesome badge: https://github.com/f/awesome-chatgpt-prompts/pull/39

Open Pull Requests I've reviewed:
- https://github.com/sindresorhus/awesome/pull/2435#pullrequestreview-1212710458
- https://github.com/sindresorhus/awesome/pull/2445#pullrequestreview-1212711880

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Review by RahulBirCodes — CHANGES_REQUESTED** [review]
The biggest problem I noticed was that there is no table of contents in the list. It's hard to navigate through the prompts without one.

**Comment by ademidun** [comment]
> The biggest problem I noticed was that there is no table of contents in the list. It's hard to navigate through the prompts without one.

Thanks fix is here: https://github.com/f/awesome-chatgpt-prompts/pull/74

**Comment by RahulBirCodes** [comment]
> > The biggest problem I noticed was that there is no table of contents in the list. It's hard to navigate through the prompts without one.
> 
> Thanks fix is here: [f/awesome-chatgpt-prompts#74](https://github.com/f/awesome-chatgpt-prompts/pull/74)

Slight issue I noticed is that there seems to be an extra space in your description about the theme of the list. Other than that it looks good!

File: readme.md [hunk 1]
```diff
@@ -650,6 +650,7 @@
 - [Product Management](https://github.com/dend/awesome-product-management#readme) - Learn how to be a better product manager.
 - [Roadmaps](https://github.com/liuchong/awesome-roadmaps#readme) - Gives you a clear route to improve your knowledge and skills.
 - [YouTubers](https://github.com/JoseDeFreitas/awesome-youtubers#readme) - Watch video tutorials from YouTubers that teach you about technology.
+- [ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts) -  Prompt examples to be used with the ChatGPT AI chat bot.
```
Comment (ademidun) [inline readme.md:653]:
```suggestion
- [ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts) - Prompt examples to be used with the ChatGPT AI chat bot.
```

**Review by ademidun — COMMENTED** [review]
@RahulBirCodes fixed extra space

**Review by sophacola — APPROVED** [review]

**Comment by Symbitic** [comment]
My advice would be to have the introduction first, then the prompts, and everything else at the end. I could feel my focus loosening as I scrolled down, especially on my phone. That's just my personal perception. If someone else wants to try it and thinks it looks easy to navigate, then I'll say it's a problem with me instead.

**Comment by stark1tty** [comment]
I feel like categorisation is needed on this list. Right now it just seems to be random. 

I'd also be interested to see a list of related awesome lists and/or useful tools. I found this list being used in the desktop app @ https://github.com/lencx/ChatGPT but it is not mentioned in this list at all.

**Comment by rvanasa** [comment]
Pointing out one of the requirements:

- Run [awesome-lint](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).

Really interesting and useful list by the way!

**Comment by epompeii** [comment]
There are several mistakes you should consider running `npx awesome-lint`.

**Review by rickstaa — APPROVED** [review]

**Review by andrzejsydor — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -650,6 +650,7 @@
 - [Product Management](https://github.com/dend/awesome-product-management#readme) - Learn how to be a better product manager.
 - [Roadmaps](https://github.com/liuchong/awesome-roadmaps#readme) - Gives you a clear route to improve your knowledge and skills.
 - [YouTubers](https://github.com/JoseDeFreitas/awesome-youtubers#readme) - Watch video tutorials from YouTubers that teach you about technology.
+- [ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts) - Prompt examples to be used with the ChatGPT AI chat bot.
```
Comment (rickstaa) [inline readme.md:653]:
```suggestion
- [ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts#readme) - Prompt examples to be used with the ChatGPT AI chat bot.
```

**Comment by rickstaa** [comment]
I think the list is very interesting. There however appear to be some problems:

>The title of your entry should be title-cased and the URL to your list should end in #readme.
Example: - [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software. 
- ❌ The description should contain the `#readme` suffix.  
> Has a Table of Contents section:
- ❌
>The repo should have awesome-list & awesome as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- ❌
>Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
> Should be placed on the right side of the readme heading.
> Can be placed centered if the list has a centered graphics header.
- ❌ Not placed at the right side of the header.
> Do not add the license name, text, or a Licence section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
- ❌ 
>Run [awesome-lint](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- ❌ The list contains 168 linting errors.

#### Other

- The list appears to contain a sales link. @sindresorhus do we allow these?

image

#### Logs

<summary>Show errors</summary>
<details>

```bash
✖ Linting

  README.md:596:2
  ✖    1:1    The repository should have "awesome" as a GitHub topic                                                                              remark-lint:awesome-github
  ✖    1:1    The repository should have "awesome-list" as a GitHub topic                                                                         remark-lint:awesome-github
  ✖    1:1    Forbidden license section found                                                                                                     remark-lint:awesome-license
  ✖    1:1    Missing or invalid Table of Contents                                                                                                remark-lint:awesome-toc
  ✖    3:157  https://www.steamship.com/build?utm_source=github&utm_medium=badge&utm_campaign=awesome_gpt_prompts&utm_id=awesome_gpt_prompts    
  remark-lint:double-link
  ✖   15:1    Don’t use emphasis to introduce a section, use a heading                                                                          
  remark-lint:no-emphasis-as-heading
  ✖   17:1    Don’t use emphasis to introduce a section, use a heading                                                                          
  remark-lint:no-emphasis-as-heading
  ✖   23:1    Don’t add a trailing ? to headings                                                                                                
  remark-lint:no-heading-punctuation
  ✖   29:1    Don’t add a trailing ? to headings                                                                                                
  remark-lint:no-heading-punctuation
  ✖   31:14   https://www.steamship.com/build?utm_source=github&utm_medium=explainer&utm_campaign=awesome_gpt_prompts&utm_id=awesome_gpt_prompts
  remark-lint:double-link
  ✖   33:9    https://www.steamship.com/build?utm_source=github&utm_medium=explainer&utm_campaign=awesome_gpt_prompts&utm_id=awesome_gpt_prompts
  remark-lint:double-link
  ✖   35:1    Don’t add a trailing ? to headings                                                                                                
  remark-lint:no-heading-punctuation
  ✖   70:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖   76:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖   82:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖   89:3    Text "javascript" should be written as "JavaScript"                                                                               
  remark-lint:awesome-spell-check
  ✖   92:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖   96:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖  118:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  122:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  123:406  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  123:456  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  126:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  130:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  134:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  138:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  142:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  146:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  150:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  154:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  163:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  167:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  168:107  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  171:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  175:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  179:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  183:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  187:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  191:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  195:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  199:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  203:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  204:397  "”" is used without matching "“"                                                                                                  
  remark-lint:match-punctuation
  ✖  207:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  211:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  215:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  219:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  223:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  227:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  231:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  235:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  239:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  243:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  247:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  251:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  255:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  259:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  260:243  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  260:349  "“" is used without matching "”"                                                                                                  
  remark-lint:match-punctuation
  ✖  263:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  264:375  "“" is used without matching "”"                                                                                                  
  remark-lint:match-punctuation
  ✖  267:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  271:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  275:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  276:558  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  279:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  280:407  "“" is used without matching "”"                                                                                                  
  remark-lint:match-punctuation
  ✖  283:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  287:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  291:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  295:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  299:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  303:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  307:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  317:1    Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  317:217  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  317:218  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  322:17   https://github.com/sinanerdinc                                                                                                    
  remark-lint:double-link
  ✖  330:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  334:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  338:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  342:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  346:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  350:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  354:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  358:17   https://github.com/iuzn                                                                                                           
  remark-lint:double-link
  ✖  359:381  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:382  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:383  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:389  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:397  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:398  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:399  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:408  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:409  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:410  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:425  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:431  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  359:432  Should not repeat "."                                                                                                             
  remark-lint:no-repeat-punctuation
  ✖  366:17   https://github.com/sinanerdinc                                                                                                    
  remark-lint:double-link
  ✖  378:17   https://github.com/f                                                                                                              
  remark-lint:double-link
  ✖  381:1    Don’t add a trailing : to headings                                                                                                
  remark-lint:no-heading-punctuation
  ✖  382:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  385:1    Don’t add a trailing : to headings                                                                                                
  remark-lint:no-heading-punctuation
  ✖  388:3    Text "StackOverflow" should be written as "Stack Overflow"                                                                        
  remark-lint:awesome-spell-check
  ✖  391:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  399:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  403:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  407:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  411:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  415:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  419:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  423:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  431:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  435:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  439:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  443:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  447:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  451:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  455:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  459:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  463:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  467:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  471:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  475:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  479:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  483:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  487:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  491:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  495:17   https://github.com/devisasari                                                                                                     
  remark-lint:double-link
  ✖  515:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  519:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  523:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  527:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  531:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  535:17   https://github.com/giorgiop                                                                                                       
  remark-lint:double-link
  ✖  539:17   https://github.com/TirendazAcademy                                                                                                
  remark-lint:double-link
  ✖  551:17   https://github.com/ersinyilmaz                                                                                                    
  remark-lint:double-link
  ✖  567:17   https://github.com/ersinyilmaz                                                                                                    
  remark-lint:double-link
  ✖  584:17   https://github.com/TirendazAcademy                                                                                                
  remark-lint:double-link
  ✖  588:4    Text "StackOverflow" should be written as "Stack Overflow"                                                                        
  remark-lint:awesome-spell-check
  ✖  591:3    Text "stackoverflow" should be written as "Stack Overflow"                                                                        
  remark-lint:awesome-spell-check
  ✖  594:17   https://github.com/ilhanaydinli                                                                                                   
  remark-lint:double-link
  ✖  596:2    Add 1 space between block quote and content                                                                                       
  remark-lint:blockquote-indentation
  ✖  599:17   https://github.com/ilhanaydinli                                                                                                   
  remark-lint:double-link
  ✖  601:2    Add 1 space between block quote and content                                                                                       
  remark-lint:blockquote-indentation
  ✖  606:2    Add 1 space between block quote and content                                                                                       
  remark-lint:blockquote-indentation
  ✖  611:2    Add 1 space between block quote and content                                                                                       
  remark-lint:blockquote-indentation
  ✖  611:617  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  624:16   https://github.com/buddylabsai                                                                                                    
  remark-lint:double-link
  ✖  631:2    Add 1 space between block quote and content                                                                                       
  remark-lint:blockquote-indentation
  ✖  631:226  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  634:17   https://github.com/buddylabsai                                                                                                    
  remark-lint:double-link
  ✖  661:169  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  661:414  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  661:698  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  666:78   Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  666:89   Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  671:320  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  671:419  "“" is used without matching "”"                                                                                                  
  remark-lint:match-punctuation
  ✖  711:250  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  716:648  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  716:789  Found reference to undefined definition                                                                                           
  remark-lint:no-undefined-references
  ✖  733:780  "’" is used without matching "‘"                                                                                                  
  remark-lint:match-punctuation
  ✖  741:17   https://github.com/y1j2x34                                                                                                        
  remark-lint:double-link
  ✖  756:16   https://github.com/y1j2x34                                                                                                        
  remark-lint:double-link
  ✖  777:1    Don’t use multiple top level headings (67:1)                                                                                      
  remark-lint:no-multiple-toplevel-headings
  ✖  777:1    List can only have one heading                                                                                                    
  remark-lint:awesome-heading

  168 errors
```

</details>

**Pull request closed without merging**